src/plugins/intel_gpu/src/graph/impls/ocl/strided_slice.cpp

```diff
@@ -31,35 +31,58 @@ struct strided_slice_impl : typed_primitive_impl_ocl<strided_slice> {
-                    sizes[j] = static_cast<int32_t>(sizes_i64[j]);
-            } else {
-                mem_lock<int32_t, mem_lock_type::read> lock{mem, arg.get_program().get_stream()};
-                int32_t* data = lock.data();
-                sizes = std::vector<int32_t>(data, data + mem->get_layout().count());
+        if (!arg.const_mem.empty()) {
+            // Getting data from constant inputs. There are 3 args: Begin, End, Stride
+            for (size_t i = 0; i < arg.const_mem.size(); ++i) {
+                auto mem = arg.const_mem[i];
+                std::vector<int32_t> sizes;
+                if (mem->get_layout().data_type == cldnn::data_types::i64) {
+                    mem_lock<int64_t, mem_lock_type::read> lock{mem, arg.get_program().get_stream()};
+                    int64_t* data = lock.data();
+                    std::vector<int64_t> sizes_i64 = std::vector<int64_t>(data, data + mem->get_layout().count());
+                    sizes.resize(sizes_i64.size());
+                    for (size_t j = 0; j < sizes.size(); j++)
+                        sizes[j] = static_cast<int32_t>(sizes_i64[j]);
+                } else {
+                    mem_lock<int32_t, mem_lock_type::read> lock{mem, arg.get_program().get_stream()};
+                    int32_t* data = lock.data();
+                    sizes = std::vector<int32_t>(data, data + mem->get_layout().count());
+                }
+                pad_vector_to_size(sizes, dims_num, i != 1);  // for "begin" completion used 0 value, for other - 1
+                params.striding_params.push_back(sizes);
+            }
+        } else {
+            // Getting data from constant inputs. There are 3 args: Begin, End, Stride
+            for (size_t i = 1; i < arg.get_dependencies().size(); ++i) {
+                auto& input = arg.get_dependency(i).as<data>();
+                auto mem = input.get_attached_memory_ptr();
+                std::vector<int32_t> sizes;
+                if (input.get_output_layout().data_type == cldnn::data_types::i64) {
+                    mem_lock<int64_t> lock{mem, arg.get_program().get_stream()};
+                    int64_t* data = lock.data();
+                    std::vector<int64_t> sizes_i64 = std::vector<int64_t>(data, data + input.get_output_layout().count());
+                    sizes.resize(sizes_i64.size());
+                    for (size_t j = 0; j < sizes.size(); j++)
+                        sizes[j] = static_cast<int32_t>(sizes_i64[j]);
+                } else {
+                    mem_lock<int32_t> lock{mem, arg.get_program().get_stream()};
+                    int32_t* data = lock.data();
+                    sizes = std::vector<int32_t>(data, data + input.get_output_layout().count());
+                }
+                pad_vector_to_size(sizes, dims_num, i != 1);  // for "begin" completion used 0 value, for other - 1
+                params.striding_params.push_back(sizes);
```

Comment:
can refactor like this? 
if (!arg.const_mem.empty()) {
    // set const mem here 
}

for size_t = 0; i < arg.const_mem.size(); ++i) {
   auto mem = arg.const_mem[i];
}

Reply:
At this point, arg(=node) and const_mem member cannot be modified because arg is passed along with const keyword.

Reply:
We can't set const mem there because we cannot use network.get_output_memory() in calc_output_layout method.
But I will try to find new way to set const_mem in stride_slice

Reply:
As discussed, this part will be handled through a separate task.